src/bin.tsx

```diff
@@ -20,9 +20,10 @@ program
   .arguments('<project-id>')
   .arguments('<access-token>')
   .option('-d --days <days>', 'older than days', '30')
-  .action((gitlabUrl: string, projectId: string, accessToken: string, options: Record<string, string>) => {
+  .option('--trace', 'show stack traces for errors when possible', false)
+  .action((gitlabUrl: string, projectId: string, accessToken: string, options: Record<string, unknown>) => {
     const parsedProjectId = parseInt(projectId, 10);
-    const days = parseInt(options.days, 10);
+    const days = typeof options.days === 'string' ? parseInt(options.days, 10) : 30;
```

Comment:
When you pass in `foo` instead of a number it was evaluated to `NaN` wich would lead to an user friendly error message here https://github.com/screendriver/gitlab-pipeline-deleter/blob/main/src/Main.tsx#L11

Reply:
Yes I know. But we don’t even know if `options.days` will be a `string`, that’s why we need to check it.